### PR TITLE
Update Descriptions in Schema

### PR DESCRIPTION
**Pull request by datamel**
Sibling PRs: https://github.com/cylc/cylc-uiserver/pull/418, https://github.com/cylc/cylc-ui/pull/1224
This enables buttons to be disabled in the ui, based on workflow state.
Closes https://github.com/cylc/cylc-flow/issues/4726, https://github.com/cylc/cylc-ui/issues/868

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` and `conda-environment.yml`.
- [x] Tests are not included (not needed, updating descriptions).
- [x] `CHANGES.md` entry not included (updating descriptions does not affect users)
- [x] Cylc-Doc PR not needed.
- [ ] If this is a bug fix, PR should be raised against the relevant `?.?.x` branch.

### CODE REVIEW

**Comment by datamel** [comment]
Yes, I will change the format. 

> And could the "Valid for" line go at the bottom of the description so that it doesn't interrupt the docs.

Happy to move it to the bottom. I moved it up so that it would be present in the short description in the ui so people could see why the buttons were disabled.

File: cylc/flow/network/schema.py [hunk 1]
```diff
@@ -1590,6 +1594,8 @@ class Meta:
             as success and failure. Applications run by jobs can use
             this command to report messages and to report registered task
             outputs.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1598]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 2]
```diff
@@ -1612,6 +1618,8 @@ class Meta:
             Release all tasks and unset the workflow hold point, if set.
 
             Held tasks do not submit their jobs even if ready to run.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1622]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 3]
```diff
@@ -1686,6 +1699,7 @@ class Meta:
             Set the maximum graph distance (n) from an active node
             of the data-store graph window.
 
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1702]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 4]
```diff
@@ -1759,6 +1775,8 @@ class Meta:
 
             Note: To manually trigger a task use "Trigger" not
             "ExtTrigger".
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1779]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 5]
```diff
@@ -1837,6 +1855,8 @@ class Meta:
             Hold tasks within a workflow.
 
             Held tasks do not submit their jobs even if ready to run.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1859]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 6]
```diff
@@ -1847,6 +1867,8 @@ class Meta:
             Release held tasks within a workflow.
 
             See also the opposite command `hold`.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1871]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 7]
```diff
@@ -1856,6 +1878,8 @@ class Kill(Mutation, TaskMutation):
     class Meta:
         description = sstrip('''
             Kill running or submitted jobs.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1882]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 8]
```diff
@@ -1871,6 +1895,8 @@ class Meta:
             Pollable tasks are those in the n=0 window with
             an associated job ID, including incomplete finished
             tasks.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1899]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 9]
```diff
@@ -1883,6 +1909,7 @@ class Meta:
         description = sstrip('''
             Remove one or more task instances from a running workflow.
 
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1912]:
```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 10]
```diff
@@ -1896,6 +1923,8 @@ class Meta:
             algorithm by artificially satisfying outputs of tasks.
 
             By default this makes tasks appear as if they succeeded.
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1927]: Technically valid for paused workflows, however, the changes won't take effect until the workflow is resumed. So there might be an argument for restricting this to just running workflows, but that could backfire by making it difficult to intervene with a workflow whilst paused.

```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 11]
```diff
@@ -1918,6 +1947,8 @@ class Meta:
             tasks will submit immediately if triggered, even if that violates
             the queue limit (so you may need to trigger a queue-limited task
             twice to get it to submit immediately).
+
+            Valid for: running workflows.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1951]: Same as above, technically valid, but nothing will happen until the workflow is resumed which could cause confusion. However, there are probably valid use cases for pausing a workflow, mucking about with the graph, then resuming it so probs best to permit it.

```suggestion
            Valid for: paused, running workflows.
```

File: cylc/flow/network/schema.py [hunk 12]
```diff
@@ -1460,7 +1460,7 @@ class Flow(String):
 class Broadcast(Mutation):
     class Meta:
         description = sstrip('''
-            Override `[runtime]` configurations in a running workflow.
+            Override `[runtime]` configurations in a workflow.
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1463]: Now the `Valid for:` bit is at the bottom the `running` here is useful again.

```suggestion
            Override `[runtime]` configurations in a running workflow.
```

File: cylc/flow/network/schema.py [hunk 13]
```diff
@@ -1639,7 +1648,7 @@ class Arguments:
 class Reload(Mutation):
     class Meta:
         description = sstrip('''
-            Reload the configuration of a running workflow.
+            Reload the configuration of a workflow.
```
Comment (datamel) [inline cylc/flow/network/schema.py:1651]: I'll put this one back so it appears in the short description.
```suggestion
            Reload the configuration of a running workflow.
```

**Review by oliver-sanders — APPROVED** [review]

**Review by wxtim — APPROVED** [review]

**Pull request merged**